worlds/seaofthieves/Regions/Regions.py

```diff
@@ -0,0 +1,88 @@
+import copy
+import typing
+
+from BaseClasses import MultiWorld, Region, Entrance
+from worlds.seaofthieves.Locations.Locations import *
+from ..Locations.LocationCollection import LocationDetailsCollection
+from ...generic.Rules import add_rule, exclusion_rules
```

Comment:
```suggestion
from worlds.generic.Rules import add_rule, exclusion_rules
```
Use absolute imports for files outside of your world. Not doing it this way breaks something on frozen I think